kitchen.yml

```diff
@@ -0,0 +1,75 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  product_name: cinc
+  enforce_idempotency: true
+  multiple_converge: 2
+  # deprecations_as_errors: true
```

Comment:
this should be uncommented

Reply:
This was commented due to `chef-vault` not enabling unified_mode. Waiting on chef-cookbooks/chef-vault#96 for that.

Reply:
@detjensrobert the feedback I got on Slack related to that PR is that `chef-vault` is not baked into Chef and we shouldn't be using that cookbook anymore. I recommend we just remove it's dep in this cookbook.